docs/testbed/README.testbed.Setup.md

```diff
@@ -219,6 +219,7 @@ Once you are in the docker container, you need to modify the testbed configurati
         ```
         ansible -m ping -i veos vm_host_1
         ```
+    - (Optional) if your environment is under a proxy to you can setup it in [`absible/group_vars/all/env.yml`](https://github.com/sonic-net/sonic-mgmt/blob/master/ansible/group_vars/all/env.yml)
```

Comment:
I feel that "environment is under a proxy" is a bit confusing. How about we just quote from the Docker docs: "If your organization uses a proxy server to connect to the internet, you may need to configure the Docker daemon to use the proxy server. You can configure it in ...."

We can also mention that we use Squid Proxy.

I think we have a typo here and below: it should be `ansible` instead of `absible`. Please fix this one and the ones down below.